app/api/items/save_grade.feature

```diff
@@ -593,9 +625,107 @@ Feature: Save grading result
       """
       {
         "data": {
-          "validated": true
+          "validated": true,
+          "unlocked_items": []
         },
         "message": "created",
         "success": true
       }
       """
+
+  Scenario: Unlocks multiple items recursively
+    Given I am the user with id "101"
+    And the database has the following table "attempts":
+      | id | participant_id |
+      | 0  | 101            |
+      | 1  | 101            |
+    And the database table "items" also has the following row:
+      | id  | platform_id | url                                                                     | validation_type | default_language_tag |
+      | 80  | 20          | http://taskplatform1.mblockelet.info/task.html?taskId=4034495436721840  | All             | fr                   |
```

Comment:
if 80 is a chapter, it is not supposed to have a url and a platform id. shouldn't it be the "type" column in `items` table definition as it changes things for the propagation?

Reply:
The propagation never looks at items.type

Reply:
Fixed the test